cfme/tests/openstack/cloud/test_provider.py

```diff
@@ -0,0 +1,40 @@
+"""Tests for Openstack cloud provider"""
+
+import pytest
+
+from widgetastic_manageiq import ParametrizedStatusBox
+
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+pytestmark = [
+    pytest.mark.usefixtures("setup_provider"),
+    pytest.mark.provider([OpenStackProvider], scope='function')
+]
+
+
+CARDS = [("Flavors", "list_flavor"), ("Images", "list_templates"),
+         ("Cloud Networks", "list_network"),
+         ("Instances", "list_vms"), ("Cloud Volumes", "list_volume")]
+
+
+@pytest.fixture(scope='function')
+def dashboard_view(provider):
+    view = navigate_to(provider, 'Details')
+    view.toolbar.view_selector.select('Dashboard View')
+    ParametrizedStatusBox(view)
```

Comment:
This is an unnecessary call to instantiate a ParametrizedStatusBox, the `ProviderDetailsDashboardView` class contains the widget already, and its being used correctly below on L32, where its instantiated with the card title.

Ultimately this doesn't really need to be a fixture, and the tests could navigate themselves - but I like the idea of reducing the duplication, especially as more tests are added for the various dashboard widgets.

Just remove this statement, everything else should work just fine.